packages/discovery-react-components/src/components/DocumentPreview/utils/box.ts

```diff
@@ -1,16 +1,27 @@
 import { CellPage } from '../types';
 import { ProcessedBbox } from '../../../utils/document/processDoc';
 
+export const LEFT = 0;
+export const TOP = 1;
+export const RIGHT = 2;
+export const BOTTOM = 3;
```

Comment:
Where are these used?

Reply:
They were used in `dom.ts`. I refactored the code to use destructuring assignment.